README.md

```diff
@@ -1,53 +1,109 @@
-# Wiki
+# Publish to GitHub wiki action
 
-This action takes the contents of a provided directory and synchronizes it with your
-repository's wiki. This allows, for example, pull requests for the wiki.
+📖 Deploy docs from your source tree to the GitHub wiki
 
-The goal is to expose the wiki to PRs, so that potential contributors who aren't
-collaborators can contribute to the wiki and go through a review process. Alternatively,
-one can disabled the "Restrict editing to collaborators only" setting for their wiki,
-but this allows the community to make unreviewed, potentially unwanted, changes.
+<div align="center">
```

Comment:
Again, I know this makes the rendered README prettier, but, for the sake of plain-text readability, I'd prefer no HTML elements. If we *really* wanted pretty documentation, we can deploy a static HTML site with GH Pages.

Reply:
> Again, I know this makes the rendered README prettier, but, for the sake of plain-text readability, I'd prefer no HTML elements. If we really wanted pretty documentation, we can deploy a static HTML site with GH Pages.

I think violating the purity in this case is OK because the benefits are greater. It's a small two lines in concession for good looking header image placement (hopefully of something more interesting than a kitten 🐱).

As for the "if we really wanted pretty, we'd use GitHub Pages", that's _extreme_ levels of customization in exchange for _more work_. And even then, you'd _still want your readme to be very good_ because some people are going to come to the readme first regardless of your fancy github pages site, and you don't get a second chance at a first impression.

It's sort of the intersection between semantic purity, editing ease/satisfaction, UX, marketing, and SEO all in one to make a good readme. **in my opinion** this minimal sacrifice of plain-text readability and purity is worth the UX of an aligned header image

but I guess sure I can drop the centering. it just looks _worse_ as a user.

Reply:
Resolved by the 3rd option of not having a header image

Reply:
I tend to very much be a purist when it comes to Markdown, for the imaginary user that clones down the repo and reads it instead of looking at it on GitHub. You could say that that my style is a bit pre-2000s, imagining users that read plain text instead of rendering stuff in the browser :laughing:

You're certainly right that centered images look better. Left-aligned images almost always look *dreadful.* Although, as an exception, I think that [this turned out alright :slightly_smiling_face:](https://github.com/spenserblack/fun-jp-terms#-fun-jp-terms)

One last thing to consider regarding HTML elements: the GitHub CLI. `gh repo view` will parse and render the Markdown in a pretty format in the terminal. How well will it handle HTML, though :thinking: